### ISSUE TITLE
Remove the flash cooy to clipboard

### ISSUE THREAD

**Issue by TheDadi** [axa-ch/style-guide#361]
Replace the flash implementation with clipboard.js — Only use flash as a fallback if the browser doesn't support the clipboard api

**Comment by davidknezic** [comment]
👍

**Comment by davidknezic** [comment]
I'd actually completely remove The Flash Way™.

**Comment by sventschui** [comment]
The copy to OS clipboard support is quiet bad. Really want to drop flash?

**Comment by davidknezic** [comment]
True. What could make our lives easier is selecting the whole thing at click and letting the users copy manually. We could do this and drop *copy to OS clipboard* altogether. What do you think?

![jtsmdgxsfj](https://cloud.githubusercontent.com/assets/198988/10508380/0d4b892e-7327-11e5-913d-35145ff7498a.gif)

**Comment by sventschui** [comment]
Wanted to implement that at first but could not find a proper design solution with anja

**Comment by anja225** [comment]
Is the lecture so boring?

**Comment by zenorocha** [comment]
+1 for clipboard.js ;)

**Comment by AndyOGo** [comment]
I just fixed the broken ZeroClipboard based implementation ...
